exercises/book-store/canonical-data.json

```diff
@@ -171,7 +171,9 @@
       ],
       "property": "total",
       "input": {
-        "basket": [1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 4, 4, 5, 5]
+        "basket": [
+          1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 4, 4, 5, 5
+        ]
```

Comment:
See #1917 why this is not desirable.

Reply:
Are you referring to:

> This makes copy and pasting harder for no reason.

If so, I don't really see why the new format is much harder, nor why copy-pasting is important here.

Reply:
> If so, I don't really see why the new format is much harder, nor why copy-pasting is important here.

Line breaks make it annoying to copy into a REPL or into the code file (which is likely `foo([...])`). It's important because it saves a lot of time when manually implementing an exercise. For tracks that use generators, the formatting is irrelevant, so I don't see why we shouldn't optimise it for human-use whenever possible. I don't see any advantage of the new format for reading or implementing the exercise, only disadvantages.

Reply:
> However. I do need to gain clarification on which exercises this idea needs to be applied to. I apologise for engaging in what-about-ism, but understanding this concern is key to making sure we are helping track maintainers and our future selves. It seems to me like book-store, bowling, change, and variable-length-quantity would fall under the same category here - that their inputs would all benefit from being on the same line as their []. That would be four exercises. Please confirm whether my understanding is correct.

Yes, that sounds right. I'm not sure if there are other exercise but those are the main ones where it makes a big difference.

Sometimes the added line breaks add readability (see Pascal's Triangle) but for a list of numbers like in these exercises, I don't see that being the case.

Reply:
> Line breaks make it annoying to copy into a REPL or into the code file
> It's important because it saves a lot of time when manually implementing an exercise.

Well, many tracks don't _have_ a REPL nor do I think that "it saves a lot of time" is necessarily true (I feel that is an assumption). 
That said, we _should_ indeed optimize for human-readability, so if it is do-able for Peter to make these exceptional cases, then I am fine with it.

Reply:
> Well, many tracks don't have a REPL nor do I think that "it saves a lot of time" is necessarily true (I feel that is an assumption).

Do those tracks gain anything from spreading it out over several lines? Maybe there are advantages, I just can't think of any.


> (I feel that is an assumption).

Try it and measure it yourself if you want, I don't see how it wouldn't be slower to remove line breaks for every paste compared to just pasting them. 🤷